extensions/VisualEditor/wikia/modules/ve/ui/pages/ve.ui.WikiaParameterPage.js

```diff
@@ -20,6 +20,15 @@ ve.ui.WikiaParameterPage = function VeUiWikiaParameterPage( parameter, name, con
 	// Parent constructor
 	ve.ui.WikiaParameterPage.super.call( this, parameter, name, config );
 
+	if (this.spec.params && this.spec.params[name] && this.spec.params[name]['type'] === 'image') {
+		// TODO display something nice
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)